### PR TITLE
Add charter and initial WG members

### PR DESCRIPTION
**Pull request by sanjacodes**
## Changes introduced with this PR

This PR adds the confirmed initial working group members, a charter, a template for decision proposals, and the record file.

Please add comments if you have any suggestions, request changes, if you'd like changes to the number of chairs or if you'd like to be added as a chair (@tsipinakis) or founding member of the Working Group (everyone else who is reading this PR).

---
By contributing to this repository, I agree to the [contribution guidelines](https://github.com/ContainerSSH/community/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: CHARTER.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+This charter covers **important processes and decisions that impact the whole organization, direction, and goals**. Small decisions like coding style per repository or similar details can be decided among the repository maintainers.
+
+## Working Group
+
+The working group *(WG)* consists of:
+* 3 chairs with equal rights who
+  - Enforce the [Code of Conduct](CODE_OF_CONDUCT.md)
+  - Have administrative rights over the GitHub org, communication channels, and other community-owned resources
+  - Publicly record decisions made by the WG in the [wg-decisions](wg-decisions) folder
+* Any individual who wants to join and passes a review of their PR sent to [WG_MEMBERS.md](WG_MEMBERS.md))
```
Comment (janosdebugs) [inline CHARTER.md:14]: I would suggest making the membership criteria more explicit:

To join the working group, you must be one of the following:

- an active user of ContainerSSH (end user, product builder, researcher, etc; please describe your use case in the PR)
- actively helping community members on any of the official ContainerSSH channels
- make code, documentation, or artwork contributions to ContainerSSH

**Review by janosdebugs — CHANGES_REQUESTED** [review]
I think the WG membership criteria should be explicit.

**Review by janosdebugs — APPROVED** [review]

**Review by bencurio — APPROVED** [review]

File: WG_MEMBERS.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+For the ContainerSSH governance model, please refer to our [charter](CHARTER.md).
+
+By adding your name to this file you confirm that you understand and agree to the [code of conduct](CODE_OF_CONDUCT.md) and [contribution guidelines](CONTRIBUTING.md).
+
+## Chairs
+
+* [Bence Santha](https://github.com/bencurio)
+* [Janos Bonic](https://github.com/janosdebugs)
+* [Sanja Bonic](https://github.com/sanjacodes)
+
```
Comment (tsipinakis) [inline WG_MEMBERS.md:12]: > if you'd like to be added as a chair (@tsipinakis)

Yes, I'd like to actively participate in this project so add me please. :)

```suggestion
* [Nikos Tsipinakis](https://github.com/tsipinakis)

```

File: CHARTER.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+# ContainerSSH Charter
+
+ContainerSSH is a **free, open source project, and must remain so**. Our main mission is to improve upon ContainerSSH with the goal to create containers for SSH users without having to set up system users.
+
+This charter covers **important processes and decisions that impact the whole organization, direction, and goals**. Small decisions like coding style per repository or similar details can be decided among the repository maintainers.
+
+## Working Group
+
+The working group *(WG)* consists of:
+* 3 chairs with equal rights who
```
Comment (tsipinakis) [inline CHARTER.md:10]:
```suggestion
* 4 chairs with equal rights who
```

**Review by tsipinakis — APPROVED** [review]

**Review by janosdebugs — APPROVED** [review]

**Comment by janosdebugs** [comment]
@tsipinakis @bencurio final review please!

**Review by bencurio — APPROVED** [review]

**Review by tsipinakis — APPROVED** [review]

**Comment by sanjacodes** [comment]
Congratulations, ContainerSSH! 🎉

**Pull request merged**